packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -65,8 +66,13 @@ export function createR11sNetworkError(
             // If a service is temporarily down or a browser resource limit is reached, RestWrapper will throw
             // a network error with no status code (e.g. err:ERR_CONN_REFUSED or err:ERR_FAILED) and
             // the error message will start with NetworkError as defined in restWrapper.ts
-            return new GenericNetworkError(
-                errorMessage, errorMessage.startsWith("NetworkError"), props);
+            // If there exists a self-signed SSL certificates error, throw a NonRetryableError
+            // Instead of relying on string matching, filter error based on the error code / ID
+
```

Comment:
```suggestion
            // TODO: instead of relying on string matching, filter error based on the error code like we do for websocket connections
```